app/services/builders/chartkick/base.rb

```diff
@@ -23,6 +23,19 @@ def build_distribution_data(entities)
         end
         entities_by_interval.sort_by { |key, _| key.slice(/[0-9]*[+-]/).to_i }
       end
+
+      def build_success_rate(entities)
+        intervals = build_distribution_data(entities)
+        total = intervals.map { |interval| interval[1] }
+                         .reduce(0) { |sum, num| sum + num }
+        return if total.zero?
+
+        successful = intervals
+                     .select { |tuple| %w[1-12 12-24].include?(tuple[0]) }
```

Comment:
`%w[1-12 12-24]` could be a constant at least? This will be configurable in the future too, right?

Reply:
That's coming on next PR 😀